CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changes by Version
 ==================
 
+0.9.0 (2017-10-25)
+------------------
+
+- Refactor trace detail [jaegertracing/jaeger-ui#53](https://github.com/jaegertracing/jaeger-ui/pull/53)
+- Virtualized scrolling for trace detail view [jaegertracing/jaeger-ui#68](https://github.com/jaegertracing/jaeger-ui/pull/68)
+- Mouseover expands truncated text to full length in left column in trace view [jaegertracing/jaeger-ui#71](https://github.com/jaegertracing/jaeger-ui/pull/71)
+- Make left column adjustable in trace detail view [jaegertracing/jaeger-ui#74](https://github.com/jaegertracing/jaeger-ui/pull/74)
+- Fix trace mini-map blurriness when < 60 spans [jaegertracing/jaeger-ui#77](https://github.com/jaegertracing/jaeger-ui/pull/77)
+- Fix Google Analytics tracking [jaegertracing/jaeger-ui#81](https://github.com/jaegertracing/jaeger-ui/pull/81)
+- Add custom menu via /api/config with project links as defaults [jaegertracing/jaeger-ui#78](https://github.com/jaegertracing/jaeger-ui/pull/78)
```

Comment:
@black-adder I recommend taking out "Add custom menu via..." because it is not supported in jaeger-query in this release.